pom.xml

```diff
@@ -54,6 +67,28 @@
             <version>42.2.27</version>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+        </dependency>
```

Comment:
I installed the plugin but it does not show any issues with jakarta-validation-api. It shows one issue with the snakeyaml but I put it on purpose because IntelliJ tells me about vulnerability in the version spring-boot-starter-web uses... Removed jakarta.validation-api dependency and the project seems to compile without issues.

Reply:
yeah, I missaid issues. I meant it helps to see where different dependencies come form. In this case we had the jakarta-validation-api dependency twice and I saw it thanks to the plugin